src/components/card/card-footer/card-footer.component.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {
+  filterStyledSystemMarginProps,
+  filterStyledSystemPaddingProps,
+} from "../../../style/utils";
+import StyledCardFooter, { StyledCardFooterProps } from "./card-footer.style";
+
+export interface CardFooterProps extends Partial<StyledCardFooterProps> {
+  /** Child nodes */
+  children: React.ReactNode;
+}
+
+const CardFooter = ({
+  spacing = "medium",
+  children,
+  transparent,
+  ...rest
+}: CardFooterProps) => {
+  return (
+    <StyledCardFooter
+      key="card-footer"
+      data-element="card-footer"
+      spacing={spacing}
+      transparent={!!transparent}
```

Comment:
`transparent` is already a boolean so doesn't need `!!`

Reply:
Actually looking into this a bit more I don't think we can go with this approach. We've missed documenting the `variant` prop in the tables but we do have an [example](https://carbon.sage.com/v/109.2.2/iframe.html?id=card--more-examples-of-card-footer&viewMode=docs#more-cardfooter--examples) of it being used so I think we will need to keep using `variant="transparent"` for now and document it in storybook